DFC.ServiceTaxonomy.GraphSync/Handlers/EventGridPublishingHandler.cs

```diff
@@ -121,6 +121,12 @@ public async Task Restored(IOrchestrationContext context)
                 return;
             }
 
+            if (context.ContentItem.Content.GraphSyncPart == null)
```

Comment:
I think it's better to stop firing this event instead  of adding condition here.

Reply:
This code is triggered by multiple events all of which would need to be modified.
Also if there was a requirement for a new event to call this code the dev would need to know to test for this condition which is something that could be missed.